game/static/game/js/game.js

```diff
@@ -692,14 +721,15 @@ ocargo.Game.prototype._setupFastTab = function () {
             }
         }.bind(this);
 
-        if (this._readyToPlay()) {
+        if (this.tabs.play.isInState('readyToPlay')) {
             flipFastSlow();
             this._resetAndPrepareAnimation(function() {
                 ocargo.animation.playAnimation();
             }.bind(this));
-        } else if (this._isPaused()) {
+        } else if (this.tabs.play.isInState('paused')) {
+            flipFastSlow();
```

Comment:
I've added this call to improve usability. Consider the following scenario:
1. Press 'Play'
2. Press 'Pause'
3. Press 'Fast'

I expect that van will start moving faster, but now it doesn't.